test/simple/test-module-loading.js

```diff
@@ -306,3 +306,18 @@ process.on('exit', function() {
 // #1440 Loading files with a byte order marker.
 assert.equal(42, require('../fixtures/utf8-bom.js'));
 assert.equal(42, require('../fixtures/utf8-bom.json'));
+
+// #9445 Error on first line of a module have the correct column number.
```

Comment:
This can just be (might need to split this across lines if it's more than 80 characters):

`// Error on the first line of a module should have the correct line and column number`